package.json

```diff
@@ -11,30 +12,33 @@
     "build:clean:launch": "yarn build && yarn clean:launch",
     "clean": "git checkout HEAD -- sandbox && git clean -f -d sandbox",
     "clean:launch": "yarn clean && yarn launch",
-    "launch": "cd sandbox && ng g @briebug/cypress-schematic:cypress",
+    "launch": "cd sandbox && ng add @briebug/cypress-schematic",
     "link:schematic": "yarn link && cd sandbox && yarn link @briebug/cypress-schematic",
     "publish": "yarn build",
     "test": "yarn build:clean:launch && yarn test:sandbox && yarn clean",
     "test:sandbox": "cd sandbox && yarn lint && yarn test && yarn build",
     "test:unit": "yarn build && jasmine src/**/*_spec.js"
   },
   "keywords": [
-    "schematics",
+    "builders",
     "cypress",
-    "testing",
     "e2e",
+    "schematics",
+    "testing",
     "time-travel"
   ],
   "author": "Briebug",
   "license": "MIT",
-  "schematics": "./src/collection.json",
+  "schematics": "./src/schematics/collection.json",
   "publishConfig": {
     "access": "public"
   },
   "dependencies": {
+    "@angular-devkit/architect": "^0.803.17",
```

Comment:
This package already comes with the CLI. It should be a peer dependency and a dev dependency.